src/pages/docs/en/basics/local.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Developing Locally
+section: Basics
+position: 3
+hidden: false
+layout: "@docs"
+---
+
+## Starting a Development Server
+
+Once you've created a Space project, you can start developing your app locally using the Space CLI's `space dev` command.
+
+```bash
+space dev # start a local development server and open the app in your browser
+```
+
+Behind the scenes, the Space CLI is:
+
+- Generating a data key to allow your app to access the Bases and Drives of your project
+- Running each of your micro in a separate process
+- Exposing all of your micros through a single entrypoint, on `localhost:4200` by default.
+
+If you want to open the app in your browser, you can use add `--open` flag:
+
+```bash
+space dev --open # start a local development server and open the app in your browser
+```
```

Comment:
I think this can be shortened:

````suggestion
If you want to open the app in your browser automatically after starting your Micros, use the `--open` flag.
````